core/controllers/profile_test.py

```diff
@@ -964,6 +964,69 @@ def test_export_account_handler(self):
                 ]
             )
 
+    def test_data_does_not_export_if_user_id_leaked(self):
```

Comment:
Is this behavior actually being tested? I don't see the export failing--it actually seems like it's succeeding. Am I maybe missing something, or is the test not being setup to trigger the failure case? I'd also expect the test to verify the error string being shown to the user.

Reply:
The export fails, but upon failure returns an error message to the user in the json. The way to validate this is via a gold zipfile, but we encountered issues with this previously https://github.com/oppia/oppia/issues/10380, so we opted to remove this.